autoload/neomake.vim

```diff
@@ -102,12 +106,14 @@ function! s:MakeJob(make_id, maker) abort
         call add(args, '%:p')
     endif
 
+    call neomake#utils#DebugMessage('Arguments before expansion: '. string(args))
     if neomake#utils#IsRunningWindows()
-        " Don't expand &shellcmdflag argument of cmd.exe
-        call map(args, 'v:val !=? &shellcmdflag ? expand(v:val) : v:val')
+        " Don't expand &shellcmdflag argument of cmd.exe nor arguments like '"something"'
+        call map(args, "(v:val ==? &shellcmdflag || v:val =~ '.*\".*\".*') ? v:val : expand(v:val)")
     else
```

Comment:
I think so.. you would have to overwrite the function in the test.
Not sure about restoring it then though - maybe store the function ref before and then assign it back?!
But if you factor out the part that is inside of `isRunningWindows` then there's no need for it.  And the surrounding function is huge IIRC anyway, so factoring it out is not too bad.